man/systemd.netdev.xml

```diff
@@ -100,6 +100,11 @@
           <row><entry><varname>gretap</varname></entry>
           <entry>A Level 2 GRE tunnel over IPv4.</entry></row>
 
+          <row><entry><varname>erspan</varname></entry>
+          <entry>ERSPAN mirrors traffic on one or more source ports and delivers the mirrored traffic to one or more destination ports on another switch.
+          The traffic is encapsulated in generic routing encapsulation (GRE) and is therefore, routable across a layer 3 network between the “source” switch
```

Comment:
"… and is therefore routable across …" (i.e. drop the comma between therefore and routable